crates/driver/openapi.yml

```diff
@@ -214,15 +244,15 @@ components:
 
             TODO: Decide what is signed and how it is signed.
           type: string
-    ExecuteRequest:
-      description: Request to the execute endpoint.
+    RevealRequest:
+      description: Request to the reveal endpoint.
       type: object
       properties:
         auctionId:
-          description: Id of the auction that should be executed.
+          description: Id of the auction that should be revealed.
           type: integer
-    ExecuteResponse:
-      description: Response of the execute endpoint.
+    RevealResponse:
```

Comment:
Renaming the `ExecuteResponse` to `RevealResponse` means that it now contains a few things that can not be known or might be irrelevant for revealing the solution (e.g. `account`, `nonce`).
Also doesn't it make sense for `/reveal` to return what is currently a `Settlement` + a signature?

Reply:
account and nonce is part of the information we need before calling execute so we can store it in the database so we later know how this auction was executed. if we didn't do this we might call /execute, crash (so we don't store anything), and the driver will submit the transaction on chain without us having recorded it.

Reply:
Hmm, I see. So I guess we then also have to think about slashing a solver when it doesn't use the account or nonce it promised? I'm not expecting this to be a common problem, though.